rmc/utils/generic.py

```diff
@@ -228,8 +228,8 @@ def get_gnomad_public_release(
     :param gnomad_data_type: gnomAD data type. Used to retrieve public release and coverage resources.
         Must be one of "exomes" or "genomes" (check is done within `public_release`).
         Default is "exomes".
-    :param adj_freq_index: Index of frequency array that contains global population calculated on
-        high quality (adj) genotypes. Default is 0.
+    :param adj_freq_index: Index of frequency array that contains frequency information calculated on
+        high quality (adj) genotypes across all genetic ancestry groups. Default is 0.
```

Comment:
Just one more suggestion for this wording (and same for the description in `mpc.py`:
```suggestion
    :param adj_freq_index: Index of array that contains allele frequency information calculated on
        high quality (adj) genotypes across genetic ancestry groups. Default is 0.
```